eclipse-scout-core/src/desktop/outline/pages/PageWithTable.ts

```diff
@@ -41,15 +45,49 @@ export class PageWithTable extends Page implements PageWithTableModel {
     this._tableDataLoadHandler = (e: TableReloadEvent) => this.loadTableData(e?.reloadReason);
   }
 
+  protected override _init(model: InitModelOf<this>) {
+    super._init(model);
+
+    // display rows as AutoLeafPageWithNodes if outline is compact
+    if (this.outline.compact) {
+      this.setAlwaysCreateChildPage(true);
+      this.setLeaf(false);
+    }
+  }
+
+  setAlwaysCreateChildPage(alwaysCreateChildPage: boolean) {
+    this.alwaysCreateChildPage = alwaysCreateChildPage;
+  }
+
   protected override _initDetailTable(table: Table) {
     super._initDetailTable(table);
+
+    if (this.outline.compact) {
+      // set table compact if outline is compact
+      table.setCompact(true);
```

Comment:
The 'moreLink' should be removed because it cannot be clicked anyway, see Table.java#createSummaryCompactHandler. Maybe htmlToPlainText is also necessary